index.js

```diff
@@ -54,6 +54,8 @@ exports.isCI = !!(
   env.CONTINUOUS_INTEGRATION || // Travis CI, Cirrus CI
   env.BUILD_NUMBER || // Jenkins, TeamCity
   env.RUN_ID || // TaskCluster, dsari
+  // Azure Pipelines
+  (env.SYSTEM_TEAMFOUNDATIONCOLLECTIONURI && env.SYSTEM_TEAMFOUNDATIONCOLLECTIONURI.startsWith('https://dev.azure.com')) ||
```

Comment:
This is just for generic environment variables used for many different CI systems. Because you added these two to `vendors.json`, the `exports.name` check will be truthy and so there's no need to add them here as well 😃